docs/admin/upgrade.rst

```diff
@@ -92,30 +90,26 @@ work, but is not as well tested as single version upgrades.
 
         weblate compress
 
-#. If you are running version from Git, you should also regenerate locale files
-   every time you are upgrading. You can do this by invoking:
+#. If you are running a Weblate version from Git, you should also regenerate locale
+   files every time you upgrade. You can do this by invoking:
 
    .. code-block:: sh
 
         weblate compilemessages
 
-#. Verify that your setup is sane (see also :ref:`production`):
+#. Verify your setup is sane (see also :ref:`production`):
 
    .. code-block:: sh
 
         weblate check --deploy
 
-#. Restart Celery worker (see :ref:`celery`).
-
+#. Restart the Celery worker (see :ref:`celery`).
 
 .. _version-specific-instructions:
 
-Version specific instructions
+Version-specific instructions
 -----------------------------
 
-.. versionchanged:: 5.0
```

Comment:
If you intended that for me, I never did. [The commit introducing it is from 14. Juli 2023](https://github.com/WeblateOrg/weblate/commit/f72cb2e40dc5238f70fba9f91068f0ab4ad38f2b), well over a year after I made my last change to this PR.

The info in question is fairly useless and introduces an error, but that is beside the point.

When you make changes [like this](https://github.com/WeblateOrg/weblate/pull/6084/commits/090d7c6791028266778247e972160cf2634e5221) it may not be obvious to you how many errors are introduced, not in the least seeing as it is an attemptedly consistent effort you are interrupting.

With the amount of non-command of English or spotty understanding of what it is that I do, it really could be better, because those are a blind-spots I no longer have, and subsequently have a harder time putting my own work through. That way everyone exposed to the end result could make the best of our combined abilities.

Instead we have no way of knowing what passes for mustard, because there is no obvious point known to be to the best of the abilities it took to make the initiative, or at least possibly good in dealing with them between you and me when you treat your own assumptions as uncritically as this.

My changes are not only are lost on you, but also the greater Weblate community, so even that feedback loop is removed.

I _really_ dislike that work I touch turn out this poorly. It tastes of disdain for how much time it takes to do.

Explaining every change and concern isn't actually fruitful for me, or anyone on the receiving end, but you can ask questions if you come up with the idea of changing any of it to be a team player.

In the years this PR was open for review, there was nothing in it that couldn't be yolo-merged.
Those are the concerns I make an effort to avoid by commenting.

The _entire_ documentation is what it is now through that process, in several passes even.
If you think about how easy it is for mistakes and errors to survive in what is now a process you have commit access to, it is impossible to attain even comparable levels of quality, much less in a sensible time-frame.

Reply:
No, it was not towards you. I needed to point it out without looking for why it happened. We discussed it with @nijel and realized it was bad it was removed.